business/tls.go

```diff
@@ -8,6 +8,7 @@ import (
 	"github.com/kiali/kiali/log"
 	"github.com/kiali/kiali/models"
 	"github.com/kiali/kiali/util/mtls"
+	core_v1 "k8s.io/api/core/v1"
```

Comment:
trivial - requires gofmt - this import needs to move up to the middle section of imports.